src/Interfaces/IDatabaseModel.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace PDGA\DataObjects\Interfaces;
+
+interface IDatabaseModel {
+    /**
+     * Get the attributes of the database model.
+     *
+     * @return array The attributes of the database model.
+     */
+    public function getAttributes(): array;
+
+    /**
+     * Get the relations of the database model.
+     *
+     * @return array The relations of the database model.
+     */
+    public function getRelations(): array;
```

Comment:
Don't the return types needed to be removed from these two methods?

Reply:
Yes, I was just waiting for a consensus and then I was working on something else.  I'll do that and close this today and open the php-api branch that uses this.